modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -678,10 +678,26 @@ void ONNXImporter::populateNet()
     CV_LOG_DEBUG(NULL, "DNN/ONNX: import completed!");
 }
 
+const std::string& extractNodeName(const opencv_onnx::NodeProto& node_proto)
+{
+    for (int i = 0; i < node_proto.output_size(); ++i)
```

Comment:
IMHO, we should try to use this value first (before outputs):

https://github.com/opencv/opencv/blob/22f0bcaf8f4fb406ae251830cf99b7ea05de40e4/modules/dnn/src/onnx/opencv-onnx.proto#L165